ConsolePlugins/EventQueueService/Main.php

```diff
@@ -67,17 +45,18 @@ public function execute(\Symfony\Component\Console\Input\InputInterface $input,
                     while (self::$run) {
                         
                         try {
-                            // Reinitialise DB
-                            $this->reinitialiseDB();
 
                             while(self::$run) {
 
                                 \Idno\Core\Idno::site()->logging()->debug('Polling queue...');
-
-                                if ($events = \Idno\Entities\AsynchronousQueuedEvent::getPendingFromQueue($queue)) {
-                                    foreach ($events as $evnt) {
+                                
+                                if ($events = \Idno\Core\Service::call('/service/queue/list/')) {
+                                    foreach ($events->queue as $event) {
                                         try {
-                                            $eventqueue->dispatch($evnt);
+                                            \Idno\Core\Idno::site()->logging()->info("Dispatching event $event");
+                                            //\Idno\Core\Service::call('/service/queue/dispatch/' . $event);
+                                            
+                                            system(escapeshellcmd("./known.php event-queue-manage $queue dispatch $event"));
```

Comment:
Not every environment can do this. It's possible to disable this ability, and many hosts do - so we need to at least check for that and fail in a way that doesn't bring Known down with a technical message. (i.e., it's got to be non-catastrophic for those users.) I don't know if that's the case here, but if it is, we need to be able to return a nice "your hosting environment doesn't support this action" sort of message.

Reply:
It only affects console services, specifically *this* console service. For now, I think the people who want or need asynchronous queues will be few. 

I'll look to improve this, but I do think a system call is going to be necessary - running in the same process is flakey.

Reply:
A nice warning is probably good enough for now though...